client/src/graphics/window.rs

```diff
@@ -229,6 +229,11 @@ impl Window {
                         VirtualKeyCode::F => {
                             down = state == ElementState::Pressed;
                         }
+                        VirtualKeyCode::V => {
+                            if state == ElementState::Pressed {
+                                self.sim.toggle_no_clip();
+                            }
+                        }
```

Comment:
nit: fold the `if` into a match guard (`VirtualKeyCode::V if state == ElementState::Pressed`)